shared/css/buttons.scss

```diff
@@ -15,6 +15,12 @@
   letter-spacing: normal;
   text-align: center;
   vertical-align: middle;
+  &[disabled=disabled],
+  &:disabled {
+    border: 1px solid $light_gray;
+    background-color: $lighter_gray;
+    box-shadow: none;
+  }
```

Comment:
This will affect disabled buttons throughout our site presumably. We're sure we're okay with that?

Reply:
This will affect disabled buttons with the `primary` class (orange buttons). I cannot find any other examples of places where we disable a primary button. There is enough going on that I can't prove we don't have any, though. Without this change, if you disable a primary button it becomes not clickable but doesn't look any different, so it looks like a bug. If we have any places where we are aware we were disabling a primary button, I think we would have fixed this already. I am fine with proceeding with this.